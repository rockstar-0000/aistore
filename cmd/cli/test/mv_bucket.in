ais wait rebalance // IGNORE
ais archive gen-shards "ais://$BUCKET_1/tmp/shard-{0..9}.tar" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais bucket mv ais://$BUCKET_1/ ais://$BUCKET_2/ --wait
ais bucket mv ais://$BUCKET_2/ ais://$BUCKET_1/
ais archive gen-shards "ais://$BUCKET_3/tmp/shard-{0..9}.tar" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais bucket mv ais://$BUCKET_1/ ais://$BUCKET_3/ // FAIL "bucket "ais://$BUCKET_3" already exists"
ais bucket mv ais://$BUCKET_1/ ais://$BUCKET_1/ // FAIL "cannot mv bucket "ais://$BUCKET_1" onto itself"

ais bucket rm ais://$BUCKET_1 ais://$BUCKET_3 // IGNORE

ais bucket mv az://bucket ais://new_bucket // FAIL "does not exist"
