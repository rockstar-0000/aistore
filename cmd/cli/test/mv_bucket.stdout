^rename-.*$
Done.
^rename-.*To monitor.*$
